### DISCUSSION TITLE
Can we change the input of flux from W/m^2 to any unit?

### DISCUSSION THREAD

**Discussion by kittipongwangnok** [phoebe-project/phoebe2#946] [General]
I just wondering about the plot of PHOEBE. Can we change the input file of flux from the unit of W/m^2 to another unit such as erg/cm^2/s/A related to the input flux?

**Comment by kecnry** [comment]
Yes, there are several ways to do this depending on what exactly you want, all of which are described in the [tutorials](https://phoebe-project.org/docs/latest/tutorials) and [API docs](https://phoebe-project.org/docs/latest/api):
* the [parameter units tutorial](https://phoebe-project.org/docs/latest/tutorials/units) shows how you can input a parameter value (including array of fluxes) in a unit other than the default parameter unit, retrieve the value in any translatable unit, as well as change the default unit
* the [API docs for b.plot](https://phoebe-project.org/docs/latest/api/phoebe.parameters.ParameterSet.plot) list `yunit` as an optional argument if you want to keep the parameter units alone but just convert the axes when plotting.

**Comment by kittipongwangnok** [comment]
Thanks, I'll check them out.

On Mon, Sep 30, 2024 at 7:04 PM Kyle Conroy ***@***.***>
wrote:

> Yes, there are several ways to do this depending on what exactly you want,
> all of which are described in the tutorials
> <https://phoebe-project.org/docs/latest/tutorials> and API docs
> <https://phoebe-project.org/docs/latest/api>:
>
>    - the parameter units tutorial
>    <https://phoebe-project.org/docs/latest/tutorials/units> shows how you
>    can input a parameter value (including array of fluxes) in a unit other
>    than the default parameter unit, retrieve the value in any translatable
>    unit, as well as change the default unit
>    - the API docs for b.plot
>    <https://phoebe-project.org/docs/latest/api/phoebe.parameters.ParameterSet.plot>
>    list yunit as an optional argument if you want to keep the parameter
>    units alone but just convert the axes when plotting.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/phoebe-project/phoebe2/discussions/946#discussioncomment-10797492>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALO4C3CAZXHOZCAJ5V5RKL3ZZE433AVCNFSM6AAAAABPCIZYQ2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZZG42DSMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Kittipong Wangnok
+6684 828 6505
School of Physics
Institute of Science
Suranaree University of Technology
